jujugui/static/gui/src/app/components/search-results/search-results.js

```diff
@@ -18,18 +18,115 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-YUI.add('search-results', function() {
+YUI.add('search-results', function(Y) {
+
+  var views = Y.namespace('juju.views'),
```

Comment:
I'd ditch this and just use Y.juju.views.Templates below as the namespace should be defined by this point or else it'll fail anyway :)